src/utils/General.ts

```diff
@@ -0,0 +1,8 @@
+/**
+   * Simple blocking sleep function
```

Comment:
well, it's still duplicated, just now duplicated into another place. Either use it from the Aquarius class, or make Aquarius class use it from this place here

Reply:
it's removed from Aquarius.ts